lib/Index/IndexDatastore.cpp

```diff
@@ -108,12 +116,16 @@ class StoreUnitRepo : public std::enable_shared_from_this<StoreUnitRepo> {
   mutable llvm::sys::Mutex StateMtx;
   std::unordered_map<IDCode, std::shared_ptr<UnitMonitor>> UnitMonitorsByCode;
 
+  llvm::StringSet<> ExplicitOutputUnitsSet;
```

Comment:
Does this need to be the names, or could it be the hashes?  This is going to be a really big set if it stores string data.

Reply:
Good point, will do that 👍